helm/aws-load-balancer-controller/README.md

```diff
@@ -163,85 +163,86 @@ Chart release v1.2.0 and later enables high availability configuration by defaul
 ## Configuration
 
 The following tables lists the configurable parameters of the chart and their default values.
-The default values set by the application itself can be confirmed [here](https://kubernetes-sigs.github.io/aws-load-balancer-controller/guide/controller/configurations/).
```

Comment:
curious if this huge diff is due to auto-formatting. If not, lets restrict the change to the newly added row.

Reply:
fixed, was some IDE auto formatting